cllc-public-app/Controllers/ApplicationsController.cs

```diff
@@ -48,6 +51,8 @@ public class ApplicationsController : ControllerBase
             _logger = loggerFactory.CreateLogger(typeof(ApplicationsController));
             _fileManagerClient = fileClient;
             _env = env;
+            _bcep = bcep;
+            _seriLogger = Serilog.Log.Logger;
```

Comment:
This is redundant.  The Serilog Logger is a static global

Reply:
The other logger is of a different type. Serilog is used by the payment controller. I made the verification method static so I can call it internally on the server( I could not pass in the other logger as is.

Reply:
You don't need to pass a logger around when using Serilog.

Code would be:

(at top of file)
using Serilog;

(and then in your code)

Log.LogError...

Reply:
I don't recommend using the Microsoft logger as we have found it has inconsistent behavior.  The Serilog logger will emit to both the console and Splunk if available.